pkg/types/date.go

```diff
@@ -5,7 +5,11 @@ import (
 	"time"
 )
 
-const DateFormat = "2006-01-02"
+var DateFormat = "2006-01-02"
```

Comment:
With this now as a `var` it'd not need a setter. I'd be considering making this unexported, and then provide a method as `DateFormat()` that returns the current value, so it's _less_ of a breaking change.

Alternatively, we just make this a `var` and no setter.